### ISSUE TITLE
im actually scared if i should download this?

### ISSUE THREAD

**Issue by plsunban** [GunsWasTakenVR/Gtag-Unbanner#2]
so that issue which said "rat" creeped me out and im wondering if hes "lying" or "right"

**Comment by Livku2** [comment]
dont

**Comment by Lollipopwfr** [comment]
rat














yes
















no












indeed














ratters













yes

**Comment by prismarinenetwork** [comment]
https://tria.ge/250202-fj4s5syjcp/behavioral1

**Comment by Gtagmodder122** [comment]
NO. You should not i tried and my pc security helped me out so do ot download this.

**Comment by GunsWasTakenVR** [comment]
> NO. You should not i tried and my pc security helped me out so do ot download this.

I don't even know why people are still replying to this crap. If someone says it's a rat why try it? I didn't make this "Unbanner" I took it from another GitHub 3 years ago. If you for some reason open it on your main pc, the webhook that sends your pc info and all of that is probably gone. (ps: if you look for gorilla tag mods/unbanners on GitHub, you're a lost cause.)

**Comment by Erik2YTreal** [comment]
rat stands 4 remote access trojan
